tests/conftest.py

```diff
@@ -0,0 +1,79 @@
+from fastapi.testclient import TestClient
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from datetime import datetime
+
+from app.main import app
+from app.database.database import Base, SessionLocal, engine
+from app.database.models import User, Event
+from app.routers import profile
+
+
+SQLALCHEMY_TEST_DATABASE_URL = "sqlite:///./test.db"
+
+test_engine = create_engine(
+    SQLALCHEMY_TEST_DATABASE_URL, connect_args={"check_same_thread": False}
+)
+TestingSessionLocal = sessionmaker(
+    autocommit=False, autoflush=False, bind=test_engine)
+
+
+def get_test_db():
+    return TestingSessionLocal()
+
+
+@pytest.fixture
+def client():
+    return TestClient(app)
+
+
+@pytest.fixture
+def session():
+    Base.metadata.create_all(bind=engine)
+    session = TestingSessionLocal()
+    yield session
+    session.close()
+    Base.metadata.drop_all(bind=engine)
+
+
+def get_test_placeholder_user():
+    return User(
+        username='fake_user',
+        email='fake@mail.fake',
+        password='123456fake',
+        full_name='FakeName'
```

Comment:
Consider using Faker instead

Reply:
The realization is taken from the master code